job_bundles/README.md

```diff
@@ -0,0 +1,277 @@
+Job submission then using the [Deadline Cloud Python package](https://github.com/aws-deadline/deadline-cloud) 
+submit your Job to Deadline Cloud. You can see the Job Bundles that are
+submitted by looking in the job history directory on your workstation after submitting a Job using a submitter plugin or the Deadline
+Cloud CLI. You can find your job history directory by running the command: `deadline config get settings.job_history_dir`.
+
+## Elements of a Job Bundle
+
+A Job Bundle is a directory structure that contains at least an
+[Open Job Description Job Template](https://github.com/OpenJobDescription/openjd-specifications/wiki) file, but may contain
+other files as follows:
+
+```
+/template.json (or template.yaml)
+/asset_references.json (or asset_references.yaml)
+/parameter_values.json (or parameter_values.yaml)
+<plus any other Job-specific files that you'd like>
+```
+
+The only required file is the Job Template (`template.json`/`template.yaml`) file that describes the structure and behaviour
+of your Job. The remaining files are optional, and are described in the following subsections.
+
+### Elements - Job Template
+
+The `template.json`/`template.yaml` file defines the runtime environment and all the processes that will run
+as part of an Amazon Deadline Cloud Job. It can be parameterized so that the same template can be used to
+create Jobs that differ only in their input values; much like a function or template in your favourite programming
+langauge. 
+
+For example, the [Job Template for the `blender_render` sample](https://github.com/aws-deadline/deadline-cloud-samples/blob/mainline/job_bundles/blender_render/template.yaml)
+defines input parameters like `BlenderSceneFile` which is a file path:
+
+```yaml
+- name: BlenderSceneFile
+  type: PATH
+  objectType: FILE
+  dataFlow: IN
+  userInterface:
+    control: CHOOSE_INPUT_FILE
+    label: Blender Scene File
+    groupLabel: Render Parameters
+    fileFilters:
+    - label: Blender Scene Files
+      patterns: ["*.blend"]
+    - label: All Files
+      patterns: ["*"]
+  description: >
+    Choose the Blender scene file you want to render. Use the 'Job Attachments' tab
+    to add textures and other files that the job needs.
+```
+
+The Job Parameter properties `userInterface`, `objectType`, and `dataFlow` are used by the Deadline Cloud CLI when
+present in a Job Template within a Job Bundle. 
+
+The `userInterface` properties control the behaviour of automatically generated Job submission UIs; both via 
+the `deadline bundle gui-submit` command line, and within Job submittion plugins for applications
+such as the [Autodesk Maya plugin](https://github.com/aws-deadline/deadline-cloud-for-maya). 
+In this example, the UI widget for inputting a value for `BlenderSceneFile` will be a file-selection dialog that
+allows filtering to see only Blender's `.blend` files or all files, and within a widget group called "Render Parameters":
```

Comment:
suggest: add a link to the GUI control showcase sample for other types of userInterface examples that people can copypasta. https://github.com/aws-deadline/deadline-cloud-samples/tree/mainline/job_bundles/gui_control_showcase